### DISCUSSION TITLE
Migrar a gitlab?

### DISCUSSION THREAD

**Discussion by ianvatega** [PucelaBits/websegura#137] [Ideas]
Se podría migrar a gitlab? En el caso de complicado se podría vincular las cuentas y que sea en modo espejo? Lo que se haga aquí se vea allí y viceversa...

por temas de que es más cercano a la filosofía GNU que GitHub ( hasta donde tengo entendido)

**Comment by adrm** [comment]
Aunque GitLab es open core, sigue sin ser software libre, así que tampoco ayuda mucho. En este caso, entiendo que pesa más el hecho de que más gente pueda colaborar aquí que en ningún otro servidor de Git. Lo bueno que tiene git en todo caso es que es descentralizado y no nos costaría nada movernos a otro servidor si fuera necesario o tuviéramos algún conflicto con GitHub. De momento lo dejaría como está.

**Comment by nukeador** [comment]
Inicialmente íbamos a crearlo en gitlab, pero los devs estaban más cómodos con los github actions para automatizar, ya que creo que en gitlab ¿tenían un coste?

**Reply by adrm** [reply]
No, creo que es bastante parecido. Lo hicimos en GitHub porque es donde Guido arrancó el repo con el primer prototipo, la verdad. Pero creo que es lo más fácil para ayudar a que la gente colabore, sencillamente porque ya lo conoce mucha gente.
